demo-app/src/main/kotlin/io/getstream/video/android/ui/join/CallJoinScreen.kt

```
@@ -195,6 +203,18 @@ fun CallJoinScreen(
+
+    if (renderFaultInjectorUi) {
+        val faultInjector = StreamVideo.instanceOrNull()?.state?.failureInjector
+        faultInjector?.let {
+            FailureInjectorUi(failureInjector = it, onClose = {
+                renderFaultInjectorUi = false
+            })
+        }
+    }
+
+    if (renderNetworkSettingsUi) {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyIfBlock)